test/e2e/failing/dune

```diff
@@ -0,0 +1,17 @@
+(include dune.inc)
+
+(rule
+ (targets dune.gen)
+ (deps (source_tree .))
+ (action
+  (with-stdout-to
+    %{targets}
+   (run %{dep:../gen_dune_rules.exe} --expect-failure)
```

Comment:
Same as for `strip_randomness.exe`, the `{%dep:` can be removed as dune interprets the first `run` argument as a dependency in this case!